resources/lang/fr.json

```diff
@@ -70,6 +70,7 @@
     "dates.July": "juillet",
     "modals.deleteStatus-title": "Est-ce que tu veux vraiment supprimer cet enregistrement ?",
     "controller.transport.social-post-with-event": "Je suis actuellement dans :lineName à destination de :destination pour #:hashtag ! #NowTräwelling | Je suis actuellement dans :lineName à destination de :destination pour #:hashtag ! #NowTräwelling ",
+    "controller.transport.social-post-for": "pour #:hashtag",
```

Comment:
I don't understand this PR. This line was already there in 44?

https://github.com/TheOneWithTheBraid/traewelling/blob/087840de31afabac2029fb9ae971a5018f134a21/resources/lang/fr.json#L44